README.md

```diff
@@ -465,17 +465,16 @@ Linux port of WinDirStat". Well, nothing could be further from the truth:
 **WinDirStat is a Windows port of KDirStat**, the predecessor of QDirStat.
 
 So it's the other way round: **The Linux version was there first**, and
-somebody liked it so much that he wrote a Windows version based on that
-idea. That's a rare thing; usually people port Windows originals to Linux.
+somebody liked it so much that they wrote a Windows version based on that
+idea.
 
 See also https://windirstat.net/background.html and the WinDirStat "About"
 dialog.
 
 
-
 ### QDirStat and K4DirStat
 
-K4DirStat is a port to KDE 4 / Qt 4 of the old KDE 3 / Qt 3 KDirStat. QDirStat is
+K4DirStat is a port to KDE 4 / Qt4 of the old KDE 3 / Qt3 KDirStat. QDirStat is
```

Comment:
Don't you see that this is even inconsistent within the same line?

KDE 4 vs. Qt4 and KDE 3 vs. Qt3?